steps/onesignal-create-push/0.0.1/step.yml

```diff
@@ -0,0 +1,69 @@
+title: |
+  OneSignal Create Push
```

Comment:
I reccomend naming the step `Send OneSignal push message` or similar, based on https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title.